bundles/org.openhab.binding.ephemeris/src/main/java/org/openhab/binding/ephemeris/internal/discovery/EphemerisDiscoveryService.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ephemeris.internal.discovery;
+
+import static org.openhab.binding.ephemeris.internal.EphemerisBindingConstants.*;
+
+import java.util.*;
+import java.util.concurrent.*;
+
+import org.eclipse.jdt.annotation.*;
+import org.openhab.core.config.discovery.*;
+import org.openhab.core.i18n.*;
+import org.openhab.core.thing.ThingUID;
+import org.osgi.service.component.annotations.*;
+import org.slf4j.*;
```

Comment:
In your last commit, you made strange changes consisting in replacing all imports by an import of * !

Reply:
I tried to switch using VSCode as IDE, I there were a bad option in the configuration.
Corrected.

Reply:
This was reverted only on certain files, not all.